server/startup/index.js

```diff
@@ -12,7 +11,6 @@ export default function () {
   Accounts();
```

Comment:
Should we move the rest of these (or maybe just `initTemplates`) to `init.js`. I feel like even though I have done it as well, having functions run in an `index.js` is sort of an anti-pattern.